app/views/qa/show.html.erb

```diff
@@ -10,18 +10,17 @@
 
   <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds">
-      <form action="<%= next_page_url %>" method="get" id="finder-qa-facet-filter-selection" class="gem-c-title" data-module="track-qa-choices">
-        <%= render "govuk_publishing_components/components/title", {
-          title: current_facet["question"],
-        } %>
-
-        <% if current_facet["description"] %>
-          <%= render "govuk_publishing_components/components/govspeak", {
-            content: current_facet["description"].html_safe
+      <form action="<%= next_page_url %>" method="get" id="finder-qa-facet-filter-selection" data-module="track-qa-choices">
```

Comment:
Can you explain in the commit message why the `gem-c-title` class was removed?

Reply:
I think it was put in by mistake. This is a class for the title gem, I don't think it was intended to be put into here.

Reply:
Probably worth noting that in the commit too 👍

Reply:
Updated 

```
The `class="gem-c-title"` from form tag as this was done in error.
```